tests/python/pants_test/backend/project_info/tasks/test_export_dep_as_jar_integration.py

```diff
@@ -118,7 +119,38 @@ def test_javac_options(self):
     self._check_compiler_options_for_target_are(target_to_test, expected_options, config)
 
   def test_extra_jvm_options(self):
-    target_to_test = f'testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options:opts'
+    target_to_test = 'testprojects/src/java/org/pantsbuild/testproject/extra_jvm_options:opts'
     config = {}
     expected_options = {'extra_jvm_options': ['-Dproperty.color=orange', '-Dproperty.size=2', '-DMyFlag', '-Xmx1m']}
     self._check_compiler_options_for_target_are(target_to_test, expected_options, config)
+
+  def test_node_module_deps_as_target_roots(self):
+    targets_to_test = [
+        'examples/src/scala/org/pantsbuild/example/several_scala_targets:greet_json',
+        'examples/src/scala/org/pantsbuild/example/several_scala_targets/node:names_to_greet',
+    ]
+    with self.temporary_workdir() as workdir:
+      pants_run = self.run_pants_with_workdir(
+        ['export-dep-as-jar'] + targets_to_test, workdir, {})
+      self.assert_success(pants_run)
+      export_output = json.loads(pants_run.stdout_data)
+      self.assertIn(
+        'examples.src.scala.org.pantsbuild.example.several_scala_targets.node.names_to_greet',
+        export_output['libraries']
+      )
+      self.assertIn(
+        '.pants.d',
+        export_output[
+          'libraries'
+        ]['examples.src.scala.org.pantsbuild.example.several_scala_targets.node.names_to_greet']['default'],
+      )
```

Comment:
super minor thing: this assert is likely an overkill, given the one following this.
another reason is that if vcfs is in use, the real path may actually not contain `.pants.d`